package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clp-ffi-js",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
@kirkrodrigues I'm not too sure about our conventions - do we need to update the version number in https://github.com/y-scope/clp-ffi-js/blob/424517334bbc81b2f2f30ae8d406b6cff52eafb0/CMakeLists.txt#L17 as well?

Reply:
I guess we should, yeah. One day we can hopefully make one depend on the other so we don't have duplicates.